lesson04/assignment/index.html

```diff
@@ -4,16 +4,97 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" href="/css/style.css">
-  <style></style>
-  <title>Document</title>
+  <link rel="stylesheet" href="css/style.css">
+  <title>hw4</title>
 </head>
 <body>
-  <div id="one">
-    <div id="two">
-
-    </div>
+  <h1>Welcome come to Jessica's Kitchen</h1>
+  <div class="auto-slide">
+    <ul class="auto-slide__content">
+      <li><img src="http://lorempixel.com/output/food-q-c-640-480-1.jpg" alt="food 1"></li>
+      <li><img src="http://lorempixel.com/output/food-q-c-640-480-2.jpg" alt="food 2"></li>
+      <li><img src="http://lorempixel.com/output/food-q-c-640-480-3.jpg" alt="food 3"></li>
+      <li><img src="http://lorempixel.com/output/food-q-c-640-480-4.jpg" alt="food 4"></li>
+      <li><img src="http://lorempixel.com/output/food-q-c-640-480-5.jpg" alt="food 5"></li>
+      <li><img src="http://lorempixel.com/output/food-q-c-640-480-6.jpg" alt="food 6"></li>
+      <li><img src="http://lorempixel.com/output/food-q-c-640-480-7.jpg" alt="food 7"></li>
+      <li><img src="http://lorempixel.com/output/food-q-c-640-480-8.jpg" alt="food 8"></li>
+    </ul>
   </div>
-
+  <section>
+      <div class="entrees" id="e1">
+          <div class="front">
+            Steam Pork Dumpling <br>
```

Comment:
Try to steer away from adding spaces with `<br>` and use padding/margin instead.